packages/did-core-test-server/generate-latest-respec-data.js

```diff
@@ -1,10 +1,52 @@
 const fs = require('fs');
+const path = require('path');
+const cheerio = require('cheerio');
+const moment = require('moment');
 const getReportResults = require('./services/getReportResults');
 const suitesInput = require('./suites/did-spec/default.json');
 
+const respecPath = path.resolve(__dirname, '../../index.html');
+
+const updateRespec = (suitesInput, suitesReportJson) => {
```

Comment:
This is some wizardry right here -- I like it -- wonder if any other W3C WG is doing stuff like this, feels like a useful design pattern.

Reply:
agreed :)